### PR TITLE
Replace travis with drone

### PR DESCRIPTION
**Pull request by mrow4a**

### CODE REVIEW

File: .drone.yml [hunk 1]
```diff
@@ -0,0 +1,153 @@
...
+
+clone:
+  git:
+    image: plugins/git:next
+
+pipeline:
+  install-server:
+    image: owncloudci/core
+    pull: true
+    version: daily-stable10-qa
```
Comment (patrickjahns) [inline .drone.yml]: either use `${OC_VERSION}` or remove the duplicate definitions in the matrix configuration

Comment (mrow4a) [reply .drone.yml]: @patrickjahns I am not sure where should I define it. Can you give me some example so I get an idea?

Comment (mrow4a) [reply .drone.yml]: @patrickjahns ping

**Comment by phil-davis** [comment]
See PR #50 that implements this with the newer drone starlark...

**Pull request closed without merging**